### ISSUE TITLE
Not working with NextCloud 12 / White page

### ISSUE THREAD

**Issue by gsantner** [alexstocker/sensorlogger#27]
I upgraded to nextcloud 12 and sensorlogger is just showing a white page now. I am on the latest release of sensorlogger.

heres a error messages found in error log:   
```
Error	PHP	Argument 6 passed to OC\NavigationManager::__construct() must implement interface OCP\IConfig, none given, called in /var/www/nextcloud/apps/sensorlogger/lib/App.php on line 19 and defined at /var/www/nextcloud/lib/private/NavigationManager.php#67	2017-05-23T13:14:59+0200
Error	PHP	Argument 5 passed to OC\NavigationManager::__construct() must implement interface OCP\IGroupManager, none given, called in /var/www/nextcloud/apps/sensorlogger/lib/App.php on line 19 and defined at /var/www/nextcloud/lib/private/NavigationManager.php#66	2017-05-23T13:14:59+0200
Error	PHP	Argument 4 passed to OC\NavigationManager::__construct() must implement interface OCP\IUserSession, none given, called in /var/www/nextcloud/apps/sensorlogger/lib/App.php on line 19 and defined at /var/www/nextcloud/lib/private/NavigationManager.php#65	2017-05-23T13:14:59+0200
Error	PHP	Argument 3 passed to OC\NavigationManager::__construct() must implement interface OCP\L10N\IFactory, none given, called in /var/www/nextcloud/apps/sensorlogger/lib/App.php on line 19 and defined at /var/www/nextcloud/lib/private/NavigationManager.php#64	2017-05-23T13:14:59+0200
Error	PHP	Argument 2 passed to OC\NavigationManager::__construct() must implement interface OCP\IURLGenerator, none given, called in /var/www/nextcloud/apps/sensorlogger/lib/App.php on line 19 and defined at /var/www/nextcloud/lib/private/NavigationManager.php#63	2017-05-23T13:14:59+0200
Error	PHP	Argument 1 passed to OC\NavigationManager::__construct() must implement interface OCP\App\IAppManager, none given, called in /var/www/nextcloud/apps/sensorlogger/lib/App.php on line 19 and defined at /var/www/nextcloud/lib/private/NavigationManager.php#62	2017-05-23T13:14:59+0200
```

E.g. the weahter app also had an error. see how they fixed it: <https://github.com/nextcloud/weather/commit/8f41dd3fd0ad824038f907a4ad51417d56fcf157>

**Comment by christianh95** [comment]
Same issue here, but instead of the white window, I get an "Internal Server Error"-Message from NC. Log messages are the same!

**Comment by alexstocker** [comment]
Thanx for your contribution.
I'll will check your bug report as soon as possible, but please be patient.

**Comment by cpranzl** [comment]
Same as christianh95 here.

**Comment by alexstocker** [comment]
Created an devel-nc branch and fixed Issue #27 there. Merge back to master-nc and update at apps.nextcloud.com still pending.

If you don't want to wait any longer, please feel free to checkout [devel-nc](https://github.com/alexstocker/sensorlogger/archive/devel-nc.zip)

Any further contributions appreciated

**Comment by alexstocker** [comment]
### Merge

Merged devel-nc to master-nc and published 0.0.4nc-hotfix at apps.nextcloud.com.

### Final testing
- install SensorLogger from apps.nextcloud.com to nextCloud12
  - Status: OK

DONE, FIXED and CLOSED
